easybuild/easyconfigs/g/Ghostscript/Ghostscript-10.02.1-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'ConfigureMake'
+
+name = 'Ghostscript'
+version = '10.02.1'
+
+homepage = 'https://ghostscript.com'
+description = """Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
+ different targets. It used to be part of the cups printing stack, but is no longer used for that."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs10021']
```

Comment:
Could still have used the style that 10.0.0 easyconfig uses,
```
source_urls = [
    'https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs%s/' % version.replace('.', ''),
]
```

Reply:
Thanks @akesandgren, I've now included this change.